migrations/Version20211217140639.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20211217140639 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema): void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->addSql('ALTER TABLE board ADD complainant_types LONGTEXT NOT NULL, ADD counterparty_types LONGTEXT NOT NULL, DROP complainant_party_types, DROP counter_party_types');
+        $this->addSql('ALTER TABLE case_entity CHANGE sorting_counterpart sorting_counterparty VARCHAR(255) DEFAULT NULL');
+    }
+
+    public function down(Schema $schema): void
+    {
+        // this down() migration is auto-generated, please modify it to your needs
+        $this->addSql('ALTER TABLE board ADD complainant_party_types LONGTEXT CHARACTER SET utf8mb4 NOT NULL COLLATE `utf8mb4_unicode_ci`, ADD counter_party_types LONGTEXT CHARACTER SET utf8mb4 NOT NULL COLLATE `utf8mb4_unicode_ci`, DROP complainant_types, DROP counterparty_types');
+        $this->addSql('ALTER TABLE case_entity CHANGE sorting_counterparty sorting_counterpart VARCHAR(255) CHARACTER SET utf8mb4 DEFAULT NULL COLLATE `utf8mb4_unicode_ci`');
+    }
+}
```

Comment:
You're allowed to change existing migrations in a pull request (before it's merged).